src/audio_core/audio_renderer.cpp

```diff
@@ -53,15 +52,15 @@ namespace {
 }
 
 [[nodiscard]] static constexpr std::tuple<s16, s16> Mix6To2WithCoefficients(
-    s16 fl_channel, s16 fr_channel, s16 fc_channel, s16 lf_channel, s16 bl_channel, s16 br_channel,
-    const std::array<float_le, 4>& coeff) {
+    s16 fl_channel, s16 fr_channel, s16 fc_channel, [[maybe_unused]] s16 lf_channel, s16 bl_channel,
```

Comment:
[[maybe_unused]] is no longer needed here since `lf_channel` is used below
```suggestion
    s16 fl_channel, s16 fr_channel, s16 fc_channel, s16 lf_channel, s16 bl_channel,
```